From aebd70b4b433aa05ec59b1d1c390efb5218c5c0e Mon Sep 17 00:00:00 2001
From: Julien Thomas <jthomas@zenetys.com>
Date: Fri, 21 Oct 2022 23:13:49 +0200
Subject: [PATCH 1/2] set sans-serif font on overlay

---
 html/src/components/terminal/overlay.ts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/html/src/components/terminal/overlay.ts b/html/src/components/terminal/overlay.ts
index 0438acf..1a8685d 100644
--- a/html/src/components/terminal/overlay.ts
+++ b/html/src/components/terminal/overlay.ts
@@ -10,6 +10,7 @@ export class OverlayAddon implements ITerminalAddon {
     constructor() {
         this.overlayNode = document.createElement('div');
         this.overlayNode.style.cssText = `border-radius: 15px;
+font-family: sans-serif;
 font-size: xx-large;
 opacity: 0.75;
 padding: 0.2em 0.5em 0.2em 0.5em;
-- 
2.21.1

